### PR TITLE
Improve bees as a service

### PR DESCRIPTION
**Pull request by kakra**
This is a series of patches to improve bees installation in the file system
and improve systemd settings. It also includes more guidance about how to work
with the config files under the service frontend starter.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ README.html: README.md
 install: ## Install bees + libs
 install: lib src test
 	install -Dm644 lib/libcrucible.so $(PREFIX)/usr/lib/libcrucible.so
-	install -Dm755 bin/bees $(PREFIX)/usr/bin/bees
+	install -Dm755 bin/bees $(PREFIX)/usr/libexec/bees
```
Comment (Nefelim4ag) [inline Makefile]: AFAIK most of the distro (Debian, Ubuntu, Arch & etc) doesn't have such path, so IMHO it's pointless to move bees

Comment (kakra) [reply Makefile]: Hmm, I'm on Gentoo... And while years back, I think I also had it in SuSE (before it became OpenSuSE). But no problem, we can stick it to sbin instead (which should still be a better place than bin). On the positive side, tho, I see that auto-completion will complete to "beesd" without be+tab=bees+d manually.

According to https://unix.stackexchange.com/questions/312146/what-is-the-purpose-of-usr-libexec other distributions would put it into /usr/lib/bees/bees instead.

**Comment by kakra** [comment]
@Nefelim4ag I added a changeset that should fit everyones needs. Are you okay with that?

**Comment by Nefelim4ag** [comment]
@kakra , you miss /bees/ from path in script,
otherwise ok.

**Comment by kakra** [comment]
@Nefelim4ag Whoops, always good to have another pair of eyes... ;-)

I changed my mind while refactoring the Makefile, that's why. If I'm not completely messed up (it was a long day), it should be okay now.

**Comment by Nefelim4ag** [comment]
@kakra , that's normal =)
I'm didn't see more problems with script and Makefile changes.

**Pull request merged**